OWNERS

```diff
@@ -1,24 +1,19 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - ahrtr            # Benjamin Wang <benjamin.ahrtr@gmail.com> <benjamin.wang@broadcom.com>
-  - fuweid           # Wei Fu <fuweid89@gmail.com>
-  - jmhbnz           # James Blair <jablair@redhat.com> <mail@jamesblair.net>
-  - serathius        # Marek Siarkowicz <siarkowicz@google.com> <marek.siarkowicz@gmail.com>
-  - spzala           # Sahdev Zala <spzala@us.ibm.com>
-reviewers:
-  - ivanvc           # Ivan Valdes <ivan@vald.es>
-  - siyuanfoundation # Siyuan Zhang <sizhang@google.com> <physicsbug@gmail.com>
+  - sig-etcd-chairs     # Defined in OWNERS_ALIASES
+  - sig-etcd-tech-leads # Defined in OWNERS_ALIASES
```

Comment:
so this is a little strange.  Normally, TLs and Chairs would be approvers and not just reviewers.

Reply:
one of the duties of TLs, for example, is to make sure that PRs get merged.  If we have TLs who are not approvers, this is very difficult.